andela/ah-frontend-space
# #166880068 Setup environment for writing tests.

#### What does this PR do?

setup an environment for writing tests

#### Description of Task to be completed?

- add jest testing framework 
- add a javascript testing utility enzyme
- add a test to check the setup functionality

#### How should this be manually tested?

- Clone this repo and switch to this branch
- `cd` into the root of the project directory and run `npm install` to install the packages
  in the package.json file.
-  Run `npm test` to check the setup functionality.
- There should be a passing test signifying success

#### What are the relevant pivotal tracker stories?

[#166880068](https://www.pivotaltracker.com/story/show/166880068)

#### Screenshots (if appropriate)

<img width="573" alt="tests" src="https://user-images.githubusercontent.com/40593659/60570980-55f26d00-9d7b-11e9-9a9c-bfd45896f6a9.png">
